.travis.yml

```
@@ -4,30 +4,24 @@ os:
   - linux
   - osx
```

Test on Windows too now
```suggestion
  - osx
  - windows
```

---

Do we care about testing this on windows?